metaserver/src/main/scala/org/langmeta/jsonrpc/MessageWriter.scala

```diff
@@ -21,31 +25,29 @@ import io.circe.syntax._
  * @note The header part is defined to be ASCII encoded, while the content part is UTF8.
  */
 class MessageWriter(out: OutputStream) extends LazyLogging {
-  private val ContentLen = "Content-Length"
-
   /** Lock protecting the output stream, so multiple writes don't mix message chunks. */
   private val lock = new Object
 
   /**
    * Write a message to the output stream. This method can be called from multiple threads,
    * but it may block waiting for other threads to finish writing.
    */
-  def write[T: Encoder](msg: T, h: Map[String, String] = Map.empty): Unit =
```

Comment:
We currently only handle the default json protocol and never used the headers param so I removed it for now